lists/malware.h

```diff
@@ -27,6 +27,22 @@ const std::vector<FilterList> malware_lists = {
     "https://disconnect.me/",
     "",
     ""
+  }), FilterList({
+    "E5484F0C-F35C-11E8-8EB2-F2801F1B9FD1",
+    "https://gitcdn.xyz/repo/NanoMeow/MDLMirror/master/hosts.txt",
```

Comment:
The underlying repo also includes a [list in the adblock format](https://github.com/NanoMeow/MDLMirror/blob/master/filter.txt) and so we should use this URL instead:
https://github.com/NanoMeow/MDLMirror/raw/master/filter.txt